.gitignore

```diff
@@ -5,3 +5,4 @@
 *~
 .project
 .pydevproject
+wsgi/media/**
```

Comment:
That folder (`cache`) is formed whenever we access the media images, but it is not needed to be tracked again again by git when pushing changes